modules/administration_manual/pages/configuration/server/background_jobs_configuration.adoc

```diff
@@ -73,7 +72,8 @@ executing:
 *  *  *  *  * /usr/bin/php -f /path/to/your/owncloud/cron.php
 ....
 
-NOTE: You have to make sure that `php` is found by `cron`, hence why we’ve deliberately added the full path to the PHP binary above (`/usr/bin/php`). On some systems it might be necessary to use *php-cli* instead of *php*.
+NOTE: You have to make sure that `php` is found by `cron`, hence why we’ve deliberately added the full path
```

Comment:
There is no need to format "php" and "cron" as inline source. Better to write them as PHP and Cron.

Reply:
? please help wat should be written here

Reply:
"NOTE: You have to make sure that PHP is found by Cron; hence why we’ve deliberately added the full path"